wezterm-gui/src/main.rs

```diff
@@ -326,7 +327,13 @@ fn run_terminal_gui(opts: StartCommand) -> anyhow::Result<()> {
 
         let cmd = if need_builder {
```

Comment:
I think doing this is a little cleaner:

```suggestion
            let config = config::configuration();
            let prog = opts.prog.iter().map(|s| s.as_os_str()).collect::<Vec<_>>();
            let mut builder = config.build_prog(if prog.is_empty() { None } else { Some(prog) })?;
            if let Some(cwd) = opts.cwd {
                builder.cwd(cwd);
            }
            Some(builder)
```